borsh-derive/src/internals/attributes/field/mod.rs

```diff
@@ -108,16 +119,13 @@ impl From<BTreeMap<Symbol, Variants>> for Attributes {
             bounds,
             serialize_with,
             deserialize_with,
+            skip: skip.is_some(),
             #[cfg(feature = "schema")]
             schema,
         }
     }
 }
 
-pub(crate) fn contains_skip(attrs: &[Attribute]) -> bool {
-    attrs.iter().any(|attr| attr.path() == SKIP)
-}
-
 #[cfg(feature = "schema")]
 pub(crate) fn filter_attrs(
     attrs: impl Iterator<Item = Attribute>,
```

Comment:
remove `attr.path() == SKIP ||` on line 133
so that `|attr| attr.path() == BORSH` becomes the filter predicate